tests/testthat/test-ggplot-area.R

```diff
@@ -3,7 +3,9 @@ context("Area")
 huron <- data.frame(year=1875:1972, level=as.vector(LakeHuron))
 huron$decade <- plyr::round_any(huron$year, 10, floor)
 
-ar <- ggplot(huron) + geom_area(aes(x=year, y=level))
+ar <- ggplot(huron) +
+  theme_grey()+
```

Comment:
I don't get the intention here (might it be that you use and abuse `git commit -a`?). If you change the original plot, I'd expect additional/corresponding tests (`expect_` statements). As a general rule, I believe you want to make an additional plot/test, not edit existing plots (unless they have become obsolete...). I guess you want to leave these two plots alone.